<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameDefDialog</class>
 <widget class="QDialog" name="GameDefDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>730</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>506</width>
    <height>730</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>506</width>
    <height>730</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Game Profile</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: #111;
}

QLabel {
	color: #FFF;
}

QToolButton {
	background-color: #222;
	color: #FFF;
	border-radius: 3px;
}

QLineEdit {
	background-color: #222;
	background: #222;
	color: #FFF;
}

QComboBox {
    border: 0px;
	background-color: #222;
	selection-background-color: #0156AB;

}

QComboBox:editable {
	background-color: #222;
	color: #1086e2;
}

QComboBox QAbstractItemView  {
    border: 0px;
    selection-background-color: #00F;
	background-color: #111;
	selection-color: #FFF;
	color: #FFF;
}

QComboBox::drop-down  {
	border: 0px;
	width: 22px;
}

QComboBox::down-arrow {
	image: url(:/images/scroll_down.png);
}

QGroupBox {
	color: #FFF;
border: 1px solid #333;
}

QScrollBar:vertical  {
    background: #111;
	border: 0;
	width: 20px;
	margin: 0 0 0 0;
	padding: 0 0 0 0;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical  {
    border: 0;
	margin: 0 0 0 0;
}

 QScrollBar::handle:vertical  {
     background: #333;
	 margin: 20px 0 20px 0;
 }
 
 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical  {
     background: none;
 }

 QScrollBar::sub-line:vertical  {
     border: 0px;
	 image: url(:/images/scroll_up.png);
     subcontrol-position: top;
     subcontrol-origin: content;
 }

 QScrollBar::add-line:vertical  {
     border: 0px;
	 image: url(:/images/scroll_down.png);
     subcontrol-position: bottom;
     subcontrol-origin: content;
 }

QPlainTextEdit {
     background-color: #222;
	border-color: #000;
	color: #FFF;
}

QToolBox::tab {
	background: #333;
	color: #FFF;
	border: 1px solid #333;
}

QToolBox::tab:selected {
	background: #555;
	color: #FFF;
	border: 1px solid #555;
}

QCheckBox {
	background-color: #111;
	color: #FFF;
}

</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>155</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Image:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnselpic">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>155</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>15</y>
     <width>401</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Title:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_image">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>96</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
border: 1px solid #333;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>670</y>
     <width>496</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(80, 80, 80);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnok">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>685</y>
     <width>100</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btncancel">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>685</y>
     <width>100</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>486</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	background-color: #111;
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabSpacing">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page_common">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>486</width>
      <height>310</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <attribute name="icon">
     <iconset resource="gde.qrc">
      <normaloff>:/images/arrow_down1.png</normaloff>:/images/arrow_down1.png</iconset>
    </attribute>
    <attribute name="label">
     <string>Common Buttons Description</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>-5</x>
       <y>0</y>
       <width>490</width>
       <height>305</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollArea {
	background-color: #111;
}

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="panel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>479</width>
        <height>487</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>479</width>
        <height>487</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>479</width>
        <height>487</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
	background-color: #111;
}</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>0</y>
         <width>460</width>
         <height>306</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vl_common">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_ps4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <attribute name="icon">
     <iconset resource="gde.qrc">
      <normaloff>:/images/arrow_down1.png</normaloff>:/images/arrow_down1.png</iconset>
    </attribute>
    <attribute name="label">
     <string>PS4 Buttons Description</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>486</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vl_ps4">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_ps3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <attribute name="icon">
     <iconset resource="gde.qrc">
      <normaloff>:/images/arrow_down1.png</normaloff>:/images/arrow_down1.png</iconset>
    </attribute>
    <attribute name="label">
     <string>PS3 Buttons Description</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>486</width>
       <height>86</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vl_ps3">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>150</y>
     <width>186</width>
     <height>76</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Consoles</string>
   </property>
   <widget class="QCheckBox" name="chk_ps3">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>40</y>
      <width>46</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PS3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_ps4">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>40</y>
      <width>46</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PS4</string>
    </property>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="edt_desc">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>401</width>
     <height>86</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>650</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>In the list above, use the checkbox to select ADS button</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edt_name</tabstop>
  <tabstop>edt_desc</tabstop>
  <tabstop>btnselpic</tabstop>
  <tabstop>chk_ps3</tabstop>
  <tabstop>chk_ps4</tabstop>
  <tabstop>btnok</tabstop>
  <tabstop>btncancel</tabstop>
 </tabstops>
 <resources>
  <include location="gde.qrc"/>
 </resources>
 <connections/>
</ui>
